float64 IMU_datax
float64 IMU_datay
float64 IMU_dataz
int8 sEMG_data